homeassistant/components/apple_tv/media_player.py

```diff
@@ -110,46 +97,32 @@ def should_poll(self):
     @property
     def state(self):
         """Return the state of the device."""
-        if not self._power.turned_on:
+        if self._manager.is_connecting:
+            return STATE_UNKNOWN
+        if self.atv is None:
             return STATE_OFF
 
         if self._playing:
 
-            state = self._playing.play_state
-            if state in (
-                atv_const.PLAY_STATE_IDLE,
-                atv_const.PLAY_STATE_NO_MEDIA,
-                atv_const.PLAY_STATE_LOADING,
-            ):
+            state = self._playing.device_state
+            if state in (DeviceState.Idle, DeviceState.Loading):
                 return STATE_IDLE
-            if state == atv_const.PLAY_STATE_PLAYING:
+            if state == DeviceState.Playing:
                 return STATE_PLAYING
-            if state in (
-                atv_const.PLAY_STATE_PAUSED,
-                atv_const.PLAY_STATE_FAST_FORWARD,
-                atv_const.PLAY_STATE_FAST_BACKWARD,
-                atv_const.PLAY_STATE_STOPPED,
-            ):
-                # Catch fast forward/backward here so "play" is default action
+            if state in (DeviceState.Paused, DeviceState.Seeking, DeviceState.Stopped):
                 return STATE_PAUSED
             return STATE_STANDBY  # Bad or unknown state?
 
     @callback
-    def playstatus_update(self, updater, playing):
+    def playstatus_update(self, _, playing):
```

Comment:
Is the second parameter `_` used/necessary?

Reply:
It is part of the callback API from `pyatv`. It is not needed here but cannot be removed due to API contract.

Reply:
Okay sounds reasonable to me.